apps/openmw/mwgui/tradewindow.cpp

```diff
@@ -356,9 +355,7 @@ namespace MWGui
         MWBase::Environment::get().getWindowManager()->getDialogueWindow()->addResponse(
             MWBase::Environment::get().getWorld()->getStore().get<ESM::GameSetting>().find("sBarterDialog5")->getString());
 
-        std::string sound = "Item Gold Up";
-        MWBase::Environment::get().getSoundManager()->playSound (sound, 1.0, 1.0);
-
+        MWBase::Environment::get().getWindowManager()->playSound("Item Gold Up");
```

Comment:
This is for grabbing gold, right? Would it make sense for grabbing gold underwater to make a different sound, since the coins would clink together in a distorted way?